### PR TITLE
Modified tests.

### PR DESCRIPTION
**Pull request by sherakama**
# Ready for review

- Fixes test to handle issue with cache.

### CODE REVIEW

File: tests/codeception/acceptance/SubThemeCest.php [hunk 1]
```diff
@@ -101,7 +101,7 @@ public function testSubTheme(AcceptanceTester $I) {
   protected function runConfigImport(AcceptanceTester $I, $disable_config_ignore = FALSE) {
     $drush_response = $I->runDrush('pm-list --filter=name=stanford_ssp --format=json');
     $drush_response = json_decode($drush_response, TRUE);
-    $saml_enabled = $drush_response['stanford_ssp']['status'] == 'Enabled';
+    $saml_enabled = ($drush_response['stanford_ssp']['status'] == 'Enabled');
```
Comment (pookmish) [inline tests/codeception/acceptance/SubThemeCest.php:104]: why the `()`? what about just a triple equal?

File: tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php [hunk 2]
```diff
@@ -315,7 +337,9 @@ public function testLockupSettingsT(AcceptanceTester $I) {
     $I->fillField('Line 4', 'Organization name');
     $I->fillField('Line 5', 'Last line full width option');
     $I->click('Save');
+    $I->see('Local Footer has been updated');
     $I->amOnPage('/');
+    $I->runDrush('cr');
```
Comment (pookmish) [inline tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php:342]: thats alot of cache clearing. the cache clearing should occur in the form submit automatically. this smells like we should do something better with the form submit. `Cache::invalidateTags(['config:system.site']);` normally does the trick

Comment (sherakama) [reply tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php:342]: It should: https://github.com/SU-SWS/stanford_profile_helper/blob/8.x-1.x/stanford_profile_helper.module#L513

But something no worky in circly

File: tests/codeception/acceptance/SubThemeCest.php (hunk 1 above)
Comment (sherakama) [reply tests/codeception/acceptance/SubThemeCest.php:104]: Just ensuring we capture the boolean.

**Comment by sherakama** [comment]
It is still failing in https://github.com/SU-SOE/soe_profile/pull/71 even with these changes.

**Pull request closed without merging**